frontend/tests/_support/Helper/Functional.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace frontend\tests\Helper;
+
+// here you can define custom actions
+// all public methods declared in helper class will be available in $I
+// note: see https://github.com/Codeception/Codeception/issues/4613
+
+class Functional extends \Codeception\Module
+{
+
+    /**
+     * Set \Codeception\Module\Yii2::$mailer to \Yii::$app->mailer from test
+     * For example
+     * public function _before(FunctionalTester $I)
+     * {
+     *     \Yii::$app->set('mailer', \frontend\fixtures\MockMailer::class);
+     *     $I->configureMailer();
+     *     $I->amOnPage(['site/contact']);
+     * }
+     */
+    public function configureMailer()
```

Comment:
mockMailer would be better name

Reply:
This function is only assign _current_  `component` `\Yii::$app->mailer` to `\Codeception\Module\Yii2::$mailer` in codeception module.

This can be a mock or real mailer